plans/artifactory-pro/hooks/run

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+echo "Starting Artifactory Pro"
+
+export JAVA_HOME=$(hab pkg path core/server-jre)
+export ARTIFACTORY_HOME={{pkg.svc_var_path}}
+
+{{pkg.svc_var_path}}/bin/artifactory.sh 2>&1
```

Comment:
@kevindickerson I think we need the hook as those exported environment variables are required in order for `artifactory.sh` to operate correctly. `JAVA_HOME` needs to be dynamically resolved at hook execution time also.